### PR TITLE
[Frontend] Feature/Add getPoolsByLiquidityProvider() and allow env configuration of websocket url

### PR DESCRIPTION
**Pull request by ryardley**
1. Adds the `getPoolsByLiquidityProvider()` function
[Pool: Connect List Pools API](https://app.asana.com/0/1198643968827245/1199227170208737) 

2. Adds configuration via env var for connecting the websocket endpoint
The following env vars now need to be set during deployment: 

* `VUE_APP_SIFNODE_API=https://somehost`
* `VUE_APP_SIFNODE_WS_API=ws://somehost:26657/websocket`

### CODE REVIEW

File: ui/core/src/api/ClpService/ClpService.ts [hunk 1]
```diff
@@ -15,12 +15,13 @@ import ReconnectingWebSocket from "reconnecting-websocket";
...
   loadAssets: () => Promise<Asset[]>;
   sifApiUrl: string;
-  nativeAsset: Asset;
+  client?: SifUnSignedClient;
 };
 
 type IClpService = {
   getPools: () => Promise<Pool[]>;
   onPoolsUpdated: (handler: PoolHandlerFn) => void;
+  getPoolsByLiquidityProvider: (address: string) => Promise<Pool[]>;
```
Comment (59023g) [inline ui/core/src/api/ClpService/ClpService.ts:24]: What's a Liquidity Provider? `eth` or `sif` address?

Comment (ryardley) [reply ui/core/src/api/ClpService/ClpService.ts:24]: sif

Comment (59023g) [reply ui/core/src/api/ClpService/ClpService.ts:24]: I see

**Comment by ryardley** [comment]
So I am just thinking this connects up the backend API but it might not be what @novelja needs exactly. We may want to surface the lp pools to the vue app via an action query? Would that work for you @novelja?

Something like:

```ts
onMount(() => {
  pools.value = actions.clp.getLiquidityProviderPools();
}) 
```

**Comment by ryardley** [comment]
So I added the action so @novelja can hook it up to the UI as above

**Review by 59023g — APPROVED** [review]

**Pull request merged**